client.py

```diff
@@ -2,15 +2,15 @@
 from vantage6.client import Client
 
 # Authenticate to the vantage6 server
-client = Client("http://127.0.0.1", 5000, "/api", log_level="debug")
-client.authenticate("org_1-admin", "password")
+client = Client("https://vantage6.local", 443, "/server/api", log_level="debug")
```

Comment:
It would be good to use env variables more to limit the number of places that need to be adjusted for our AWS deployment